live-build/misc/ansible-roles/appliance-build.dcenter/tasks/ldap.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+
+# Adapted from:
+#   * https://help.ubuntu.com/community/AutofsLDAP
+#   * https://www.digitalocean.com/community/tutorials/how-to-authenticate-client-computers-using-ldap-on-an-ubuntu-12-04-vps
+
+- name: Ubuntu Linux LDAP Access | Install LDAP OS Packages
```

Comment:
please remove the "name" from here, and below tasks to be consistent with our other roles/tasks.

Reply:
I am against removing the name here as it provides useful comments. It will be harder for whoever decides to update the names if they are not familiar with the code, so I feel like we should just leave the names here and update it elsewhere whenever we have time for it. I don't feel like consistency is as important (also debugging without names right now is definitely more tedious).

Reply:
If we want to use names, then please add a name to all existing tasks and update the ansiblelint configuration to enforce the use of names moving forward (perhaps in a separate/isolated commit). It'll be good to come up with the convention we want to use for the names; personally, I find the `foo bar | blah blah` convention weird, since the output of ansible already tells us the name of the role for that task being run (the `foo bar` part just seems like needless duplication.

With that said, I rather just not use names altogether, and get ansiblelint to complain when a name is used. I've never found them useful.

Reply:
Well, looks like maybe it's just me who finds them useful. I find they provide quick comments on what's being done without needing explicit comments. That said, if we are moving away from using names all-together then it'd make sense not to have names here. It's also possible that the way that we were using some roles made it harder to follow what's failing (e.g. the install-delphix-s3-packages role that is reference by several other roles), but since we are getting rid of it maybe it's fine.